src/commands/rep.ts

```diff
@@ -0,0 +1,62 @@
+import Command from ".";
+
+import type { PermissionString, CommandInteraction } from 'discord.js';
+
+export default class RepCommand extends Command {
+  name = 'rep';
+  description = 'Either give a good or a bad reputation point to an user.';
```

Comment:
I think there's improvements to be made on this, perhaps just simplifying it to:
```suggestion
  description = 'Give the donator of your last won giveaway a reputation point.';
```